docs/self-managed/identity/user-guide/configuration/configure-logging.md

```diff
@@ -0,0 +1,116 @@
+
+### Supported logging outputs
+
+As part of configuration Identity provides multiple appenders for outputting logs, to configure which logging appender
+is
+used, set the `IDENTITY_LOG_APPENDER` environment variable to one of the following `Console`, `Stackdriver`, or `Log`:
+
+<Tabs groupId="loggingAppenders" defaultValue="console"
+values={[{label: 'Console', value: 'console', }, {label: 'Stackdriver', value: 'stackdriver', }, {label: 'File', value: 'file', },]} >
+<TabItem value="console">
+
+Console logging produces messages to standard output and is the default log appender. The Console log appender offers
+additional
+configuration options, these are:
+
+| Environment variable   | Accepted values                                                                                                                           |
+| ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| `IDENTITY_LOG_PATTERN` | _See the [Log4j2 pattern layout docs](https://logging.apache.org/log4j/2.x/manual/layouts.html#PatternLayout) for possible placeholders._ |
+
+</TabItem>
+<TabItem value="stackdriver">
+
+The Stackdriver log appender produces messages to standard output in a format that is compatible with the GCP cloud
+platform.
+
+This appender uses
+the [GCP layout](https://github.com/apache/logging-log4j2/blob/2.x/log4j-layout-template-json/src/main/resources/GcpLayout.json)
+provided by the [Log4j2](https://logging.apache.org/log4j/2.x/manual/) library.
+
+</TabItem>
+<TabItem value="file">
+
+The File log appender produces messages to a rotating log file. The File log appender offers additional configuration
+options, these are:
+
+| Environment variable              | Accepted values                                                                                                                                                          |
+| --------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `IDENTITY_LOG_FILE_PATTERN`       | _See the [Log4j2 pattern layout docs](https://logging.apache.org/log4j/2.x/manual/layouts.html#PatternLayout) for possible placeholders._                                |
+| `IDENTITY_LOG_FILE_ROTATION_DAYS` | _See the [Log4j2 time-based triggering policy -> interval](https://logging.apache.org/log4j/2.x/manual/appenders.html#timebased-triggering-policy) for possible values._ |
+| `IDENTITY_LOG_FILE_ROTATION_SIZE` | _See the [Log4j2 size-bsed triggering policy](https://logging.apache.org/log4j/2.x/manual/appenders.html#sizebased-triggering-policy) for possible values._              |
+
+</TabItem>
+</Tabs>
+
+### Providing your own logging configuration
+
+The default logging configuration file can be overridden by setting the following variable:
```

Comment:
This sentence just tells the user to set the `LOGGING_CONFIG`, which does not answer following questions that users will probably have:
- where is my logging configuration?
- how do I add my logging configuration to Identity?
- are there any rules for file location?

May be something like?

```
You can provide your own configuration by mounting the configuration file to the Identity container and setting it's path in the following variable:
```